# HG changeset patch
# User Ben Abbott  <bpabbott@mac.com>
# Date 1293661776 18000
#      Wed Dec 29 17:29:36 2010 -0500
# Node ID bc509d5f763f1b8794ead09dabf2d0ab18f9770d
# Parent  00b0aa8d539e3d23a6a4d20827c07a5e4b7ec219
text.m: New demo for 3D plot and text with verticalalignment.

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,10 +1,11 @@
 2010-12-29  Ben Abbott  <bpabbott@mac.com>
 
+	* plot/text.m: New demo for 3D plot and text with verticalalignment.
 	* plot/__go_draw_axes__.m: Enable vertical alignment of text objects for
 	gnuplot.
 
 2010-12-28  Ben Abbott <bpabbott@mac.com>
 
 	* plot/private/__plt__.m: Fix to allow inline legend keys. Bug 31991.
 
 2010-12-28  Rik  <octave@nomad.inbox5.com>
diff --git a/scripts/plot/text.m b/scripts/plot/text.m
--- a/scripts/plot/text.m
+++ b/scripts/plot/text.m
@@ -123,8 +123,22 @@ endfunction
 %! set (gca, "xtick", [0.25, 0.5, 0.75],
 %!           "xticklabel", ha,
 %!           "ytick", [0.25, 0.5, 0.75],
 %!           "yticklabel", va)
 %! axis ([0 1 0 1])
 %! xlabel ("horizontal alignment")
 %! ylabel ("vertical alignment")
 %! title ("text alignment and rotation (0:30:360 degrees)")
+
+%!demo
+%! clf
+%! h = mesh (peaks, "edgecolor", 0.7 * [1 1 1],
+%!                  "facecolor", "none",
+%!                  "facealpha", 0);
+%! for t = 0:45:359;
+%!   text (25, 25, 0, "Vertical Alignment = Bottom",
+%!                    "rotation", t,
+%!                    "horizontalalignment", "left",
+%!                    "verticalalignment", "bottom")
+%! endfor
+%! caxis ([-100 100])
+%! title ("Vertically Aligned at Bottom")
